components/button/IconTextButton.module.scss

```diff
@@ -0,0 +1,27 @@
+@import 'styles/mixins';
+
+.iconTextButton {
+  @include centralize;
+  @include font(8px, 12px, 400);
+  background: transparent center / contain no-repeat;
+  border: 0;
+  color: var(--color-primary-50);
+  column-gap: 6px;
+  cursor: pointer;
+  display: inline-flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  height: 42px;
+  outline: 0;
+  padding: 0;
+  row-gap: 4px;
```

Comment:
Please, verify this and column-gap works in Safari.

Reply:
Replaced `row-gap` with `gap`.